.github/workflows/test-action.yml

```diff
@@ -0,0 +1,28 @@
+name: Test generate-metadata-action
+
+on:
+  push:
+    branches:
+      - "*"
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  action-run:
+    runs-on: ubuntu-latest
+    outputs:
+      filepath: ${{ steps.action-run.outputs.filepath }}
+    steps:
+      - name: 'Checkout directory'
+        uses: actions/checkout@v2
+      - name: Generate metadata file
```

Comment:
To test this action when changes are pushed to PR branches, I think it we'd need to just access the action locally after the checkout step with `- uses: ./ ` similar to https://github.com/hashicorp/action-setup-waypoint/blob/b6de6103b4725021981ffdf9b190d00226d71b71/.github/workflows/test.yml#L22. Otherwise we aren't really testing the new changes until the code is merged to main

Reply:
Thank you for the comment, not sure I understand why the `- uses ./` step is needed. In order to test it, first, this PR needs to be merged  [as per this line](https://github.com/hashicorp/actions-generate-metadata/blob/action_files/.github/workflows/test-action.yml#L21)

I did try the recommended [here is the link to the build](https://github.com/hashicorp/actions-generate-metadata/actions/runs/1309082410), not sure what is expected here. Lets chat offline 

There is a test setup for this action ( I left the link in the slack thread, as its not public)